React/Modules/RCTNativeAnimatedModule.m

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+#import "RCTNativeAnimatedModule.h"
+#import "RCTBridge.h"
+#import "RCTNativeAnimationManager.h"
+
+@implementation RCTNativeAnimatedModule
+
+@synthesize bridge = _bridge;
+
+RCT_EXPORT_MODULE(NativeAnimatedModule)
+
+- (instancetype)init
```

Comment:
It's preferable not use override init on modules unless you really need to, as it causes them to be eagerly initialized when bridge starts.

Could `[[RCTNativeAnimationManager sharedManager] setNativeAnimationModule:self];` be called lazily the first time one of RCTNativeAnimatedModule methods is called instead?